plugins/feeds/public/benkow_rat.py

```diff
@@ -0,0 +1,61 @@
+import requests
+from datetime import datetime, timedelta
+import csv
+import logging
+
+from core.observables import Url, Ip
+from core.feed import Feed
+from core.errors import ObservableValidationError
+from core.config.config import yeti_config
+
+class BenkowTrackerRat(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=12),
+        "name": "BenkowTrackerRat",
+        "source" : "http://benkow.cc/export_rat.php",
+        "description": "This feed contains known Malware C2 servers",
+    }
+
+    def update(self):
+        resp = requests.get(self.source, proxies=yeti_config.proxy)
+        if resp.ok:
+            reader = csv.reader(resp.content.strip().splitlines(), delimiter=';', quotechar='"')
+            for line in reader:
+                self.analyze(line)
+
+    def analyze(self, line):
+        if line[0] == 'id':
+            return
+
+        ID, Family, _Url, _IP, First_Seen , _blank = line
+        if "://" not in _Url:
```

Comment:
Please user `startswith`, and also consider the case of HTTPS. 

Otherwise, URLs added without the leading `http://` will automatically be "corrected" by Yeti upon insertion.

Reply:
> Otherwise, URLs added without the leading http:// will automatically be "corrected" by Yeti upon insertion.

False :( 
that doesn't work

Reply:
added both checks, we need to figurate why it not prepend `http(s)`